crates/refunder/src/ethflow_order.rs

```diff
@@ -0,0 +1,110 @@
+use database::{ethflow_orders::EthOrderPlacement, orders::Order};
+use ethcontract::{Bytes, H160, U256};
+use number_conversions::big_decimal_to_u256;
+// Data structure reflecting the contract ethflow order
+// https://github.com/cowprotocol/ethflowcontract/blob/main/src/libraries/EthFlowOrder.sol#L19
+pub struct EthflowOrder {
+    buy_token: H160,
+    receiver: H160,
+    sell_amount: U256,
+    buy_amount: U256,
+    app_data: Bytes<[u8; 32]>,
+    fee_amount: U256,
+    valid_to: u32,
+    flags: bool,
```

Comment:
Nit: this could be `partially_fillable` since there are no other flags available.